src/fastoad/notebooks/02_explanation/04_OpenMDAO.ipynb

```diff
@@ -0,0 +1,892 @@
+{
```

Comment:
<blockquote>The default value for any input/output, when not specified is 1.0, which is fixed by OpenMDAO.&nbsp;</blockquote><p>--&gt; "When not specified, the default value for any input/output is set to 1.0 by OpenMDAO" (looks more readable to me)</p><p> </p><blockquote>This enables to track if either a previous component that was supposed to compute said input is missing or if we did not fill it properly in the data file.</blockquote><p>--&gt; "This enables to track if an input has not been set in data file nor in other components." (globally, I'd like to reduce verbosity)</p><p> </p><blockquote>The default value can <strong>hower</strong> be overwritten by something else,</blockquote><p>"However" ?</p><p> </p><blockquote>&nbsp;In FAST-OAD(-GA) however it is not possible to define it afterwards and must be done when declaring the variable as output.</blockquote><p>--&gt; "In FAST-OAD(-GA) however<code>,</code> it is not possible" (This is my last remark about missing commas for this review, but I'll come back for that on round 2, so please, try to fix them all. It really helps the reader)</p><p> </p><blockquote>Remark (for more advanced users)</blockquote><blockquote>On a very similar subject, OpenMDAO allows to...</blockquote><p>How is it so "very similar"? But more importantly, wouldn't it be better to simply mention this option feature and send the reader to OpenMDAO documentation ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Changed in f33eb885

For the options however, I get that it extends an already long notebook, but I do think that it can be useful, especially since it's not uncommon in FAST-OAD_CS25 and FAST-OAD-GA

Reply:
<p>OK for me.</p><p> </p><p>2 minor modifications:</p><blockquote>For future users, a more detailed explanation of the variable naming in FAST-OAD is available&nbsp;<a href="https://fast-oad.readthedocs.io/en/latest/documentation/custom_modules/add_modules.html?highlight=variable%20naming#variable-naming" target="_blank">here</a><code>.</code></blockquote><p> </p><p>and I would remove "more" from</p><blockquote><strong>Remark (for </strong><code><strong>more</strong></code><strong> advanced users)</strong></blockquote>

Reply:
Changed in 3a141cc1